src/Tests/RegistrationTest.php

```diff
@@ -38,6 +65,21 @@ public function testNormalNone() {
    */
   public function testAcquisitionNone() {
```

Comment:
If we abstract all the bits mentioned above, this basically becomes calls to those methods I think...